qa-pipelines/tasks/lib/cf-deploy-upgrade-common.sh

```diff
@@ -26,7 +26,19 @@ fi
 # Check that the kube of the cluster is reasonable
 bash ${CAP_DIRECTORY}/kube-ready-state-check.sh kube
 
-PROVISIONER=$(kubectl get storageclasses persistent -o "jsonpath={.provisioner}")
+if kubectl get sc | grep "persistent" > /dev/null ; then
```

Comment:
prefer full names (`storageclass` vs `sc`) for readability

Reply:
The storageclass name used should always be `persistent`. If it has a different name in EKS, you can create a new sc named persistent with the same definition of whatever sc you want to use.

Reply:
That can be achieved once we have a deploy-eks script. But right now I have made this script in a way that would let QA use their EKS clusters. Every SUSE doc related to CAP (including Stratos and Metrics) refers to `gp2` as sc.
Also, I don't think we need to follow `storageclass name used should always be persistent`. Its a configurable parameter.

Reply:
EKS CI needs to support all EKS cluster created by referring just SUSE docs. 
Once we have `deploy-eks` script/terraform, we can then take a position that QA CI only works with clusters created by our script. Then we can totally assume that SC is going to be named `persistent`, but until them we will have to make CI compatible with sc name `gp2` as this is what our docs say.